backend/app/processing/kb.py

```
@@ -265,37 +265,54 @@ def search_kb(chatbot_id: str, query: str, k: int = 5) -> List[str]:
-                if fname.startswith('.') or fname.startswith('~$'):
-                    continue
-                full_path = os.path.join(root, fname)
-                # Only include supported extensions
-                if any(full_path.lower().endswith(ext) for ext in SUPPORTED_EXTENSIONS):
-                    file_paths.append(full_path)
-    
+
+    file_paths = collect_supported_files([kb_dir, scrapped_dir])
+
     if not file_paths:
         logger.warning(f"No supported files found for chatbot {chatbot_id}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)